deployments/kubernetes/dev/README.MD

```diff
@@ -0,0 +1 @@
+# Kustomize Merak-Compute test develeopment
```

Comment:
@phudtran same here, add some sentence to describe that this folder contains and what are they use for.

Reply:
Ok, I've updated the README